lib/versions.js

```diff
@@ -1,16 +1,25 @@
 const semver = require('semver')
 
-module.exports.getVersionInfo = lastRelease => {
+module.exports.getVersionInfo = (lastRelease, patchVersions = true) => {
   let lastVersion =
     semver.coerce(lastRelease.tag_name) || semver.coerce(lastRelease.name)
 
   if (!lastVersion) {
     return undefined
   }
 
-  return {
+  let versionInfo = {
     incrementedMajor: semver.inc(lastVersion, 'major', true),
     incrementedMinor: semver.inc(lastVersion, 'minor', true),
     incrementedPatch: semver.inc(lastVersion, 'patch', true)
   }
+
+  if (patchVersions === false) {
+    delete versionInfo.incrementedPatch
+    Object.keys(versionInfo).forEach(function(key) {
+      versionInfo[key] = versionInfo[key].replace(/\.\d+$/, '')
```

Comment:
Do you think it'd be more clear if we extract the regex to a named variable or adding a comment would make it a bit more clear what's going on this line? Regexes are always difficult to parse in my brain :)

Reply:
My brain mostly thinks regex 😭 I can replace it with substring

```suggestion
      let index = versionInfo[key].lastIndexOf('.')
      versionInfo[key] = versionInfo[key].slice(0, index)
```

Reply:
Both is fine. But in the case of regex it'd be nice if the regex had a proper name like `const REMOVE_LAST_VERSION_SECTION` (not sure if this is a very good name) so that the code can be scanned. I had to put the regex in regexr.com to figure out what it was doing now :)